deployment/helm/templates/deployment.yaml

```
@@ -69,10 +69,8 @@ spec:
           - "--github-app-client-id-file=/secrets/github-app/github_app_client_id"
           - "--github-app-client-secret-file=/secrets/github-app/github_app_client_secret"
           env:
-          - name: "MINDER_KEY_PATH"
+          - name: "MINDER_CRYPTO_KEYSTORE_LOCAL_KEYDIR"
```

Underscores are retained in the env variable
```suggestion
          - name: "MINDER_CRYPTO_KEYSTORE_LOCAL_KEY_DIR"
```